core/wiki/macros/translink.tid

```diff
@@ -2,14 +2,30 @@ title: $:/core/macros/translink
 tags: $:/tags/Macro
 
 \define translink(title,mode:"block")
-<div style="border:1px solid #ccc; padding: 0.5em; background: black; foreground; white;">
-<$link to="""$title$""">
-<$text text="""$title$"""/>
+<div class="tc-translink-outer">
+<$link to=<<__title__>>>
+<$text text=<<__title__>>/>
 </$link>
-<div style="border:1px solid #ccc; padding: 0.5em; background: white; foreground; black;">
-<$transclude tiddler="""$title$""" mode="$mode$">
-"<$text text="""$title$"""/>" is missing
+<$set name="state" value=<<current-widget-id "$:/state/translink">>>
+<span style="float: right;">
+<$reveal state=<<state>> type="nomatch" text="show">
+<$button class=<<tv-config-toolbar-class>> set=<<state>> setTo="show">
```

Comment:
hardcoded texts as this are not possible in the core. It needs to be language specific. Eg: new text in Buttons.multids

Reply:
there is no hardcoded text as far as I see @pmario

Reply:
... aaah now I've found it, well ther is hardcoded text and It's already been there

Reply:
sorry. I did add the comment to the wrong place. It's line 25. the "is missing" text

Reply:
I've now changed it to display the "Missing tiddler ViewTemplate" if the tiddler is missing. That should solve the problem since it's multilingual